src/models/metadata.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const config = require('../services/config')
+
+function getMetadata () {
+  return {
+    public_key: config.getIn(['keys', 'ed25519', 'public']),
+    urls: {
```

Comment:
Well this whole pull request is refactoring. It's a little strange to move something if you intend to move it right back.